### DISCUSSION TITLE
Improve user experience of auto hiding nav bar

### DISCUSSION THREAD

**Discussion by 117649** [black7375/Firefox-UI-Fix#803] [Ideas]
![image](https://github.com/black7375/Firefox-UI-Fix/assets/8281345/0908c7e4-7b5b-451d-8d73-d7b5224eec92)
Is it possible to leave some marks outside to mark the position of the items in nav bar while the bar is hidden?
Like this:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/8281345/a3309d6c-1753-416a-8418-76abec2f5934)
This would be very helpful when you want to locate the button you want fast while using the auto hiding nav bar.

**Comment by black7375** [comment]
How can I mark naturally?
Have you ever thought about it?

**Reply by 117649** [reply]
A floating transparent mini icon of the actual button would be most natural for me.
But if a dot is all that can be done I'm also OK with it.
